<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lehelugeja</class>
 <widget class="QDialog" name="Lehelugeja">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Finaal</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Pordi nimi:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="portBox">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>COM1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM5</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Relv:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="relvBox">
     <item>
      <property name="text">
       <string>Õhupüss</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Õhupüstol</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sportpüss</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Ühenduse logi:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTextEdit" name="logiAken">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="uhendaNupp">
     <property name="text">
      <string>Ühenda</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="seadistaNupp">
     <property name="text">
      <string>Seadista</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="lopetaNupp">
     <property name="text">
      <string>Lõpeta</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="loobuNupp">
     <property name="text">
      <string>Sulge</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>loobuNupp</sender>
   <signal>clicked()</signal>
   <receiver>Lehelugeja</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
